.evergreen/config.yml

```diff
@@ -1377,39 +1377,47 @@ tasks:
     commands:
       - func: run-make
         vars:
-          targets: "build"
+          targets: "build build-tests"
           BUILD_ENV: "PATH=/opt/golang/go1.9/bin:$PATH GOROOT=/opt/golang/go1.9"
 
   - name: go1.10-build
     tags: ["compile-check"]
     commands:
       - func: run-make
         vars:
-          targets: "build"
+          targets: "build build-tests"
           BUILD_ENV: "PATH=/opt/golang/go1.10/bin:$PATH GOROOT=/opt/golang/go1.10"
 
+  # Build with whatever the latest Go version is that we're using for tests
```

Comment:
This was added because we will have tests that are build-flagged by Go versions, so they won't be covered by the previous 1.9 and 1.10 build tasks.